### PR TITLE
Fix 'unresolved doc reference' for material A-D

### PR DESCRIPTION
**Pull request by goderbauer**
https://github.com/flutter/flutter/issues/31931

### CODE REVIEW

**Review by HansMuller — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/material/animated_icons/animated_icons.dart [hunk 1]
```diff
@@ -87,11 +87,6 @@ class AnimatedIcon extends StatelessWidget {
   ///
   /// Announced in accessibility modes (e.g TalkBack/VoiceOver).
   /// This label does not show in the UI.
-  ///
-  /// See also:
-  ///
-  ///  * [Semantics.label], which is set to [semanticLabel] in the underlying
```
Comment (tvolkert) [inline packages/flutter/lib/src/material/animated_icons/animated_icons.dart:93]: '[SemanticProperties.label]'?

File: packages/flutter/lib/src/material/bottom_app_bar.dart [hunk 2]
```diff
@@ -82,14 +83,16 @@ class BottomAppBar extends StatefulWidget {
   /// This controls the size of the shadow below the bottom app bar. The
   /// value is non-negative.
   ///
-  /// If this property is null then [ThemeData.bottomAppBarTheme.elevation] is used,
-  /// if that's null, the default value is 8.
+  /// If this property is null then [BottomAppBarTheme.elevation] of
+  /// [ThemeData.bottomAppBarTheme] is used, if that's null, the default value
```
Comment (tvolkert) [inline packages/flutter/lib/src/material/bottom_app_bar.dart:87]: Nit: `s/used, if/used. If/`

Comment (tvolkert) [reply packages/flutter/lib/src/material/bottom_app_bar.dart:87]: (and elsewhere in this file)

File: packages/flutter/lib/src/material/theme_data.dart [hunk 3]
```diff
@@ -77,7 +77,6 @@ const Color _kDarkThemeSplashColor = Color(0x40CCCCCC);
 ///   * [OutlineButton]
 ///   * [FlatButton]
 ///   * [RaisedButton]
-///   * [TimePicker]
```
Comment (tvolkert) [inline packages/flutter/lib/src/material/theme_data.dart:80]: `* The time picker widget ([showTimePicker])` ?

**Review by tvolkert — APPROVED** [review]

**Pull request merged**